en/setup/building_px4.md

```diff
@@ -394,8 +394,8 @@ Before starting Qt Creator, the [project file](https://cmake.org/Wiki/CMake_Gene
 
 ```sh
 cd ~/src/Firmware
-mkdir build_creator
-cd build_creator
+mkdir -p build/creator
+cd build/creator
 cmake .. -G "CodeBlocks - Unix Makefiles"
```

Comment:
Should be `cmake ../.. -G ...`

Same in FW Makefile: https://github.com/PX4/Firmware/blob/master/Makefile#L339